docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
+  zookeeper:
+    image: 'wurstmeister/zookeeper:latest'
```

Comment:
Let's pin these to specific versions so new releases don't break this setup.

Zk - 3.6
Kafka: 2.8
Grafana: 7/8

Reply:
Switched from `wurstmeister` to `bitnami` images as they had slightly newer versions, and much better tagging. Otherwise they should be effectively identical for our use case.